app/models/user.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        defaultValue: sequelize.UUIDV1,
+        field: 'id'
```

Comment:
hint, when the field is called the same in the database you don't need to add the field field